src/core/favorites-state/favorite-state.service.ts

```diff
@@ -25,4 +28,8 @@ export class FavoriteStateService {
   isFavoriteCity(city: string): boolean {
     return this.storage.getItem(this.FAVORITES_KEY).indexOf(city) === -1;
   }
+
+  moreCitiesCanBeAdded(): boolean{
+    return (this.storage.getItem(this.FAVORITES_KEY).length > this.MAX_AMOUNT_OF_FAVS) ? false : true;
```

Comment:
just this.storage.getItem(this.FAVORITES_KEY).length > this.MAX_AMOUNT_OF_FAVS

Reply:
fixed method